octobot_commons/constants.py

```diff
@@ -179,7 +180,7 @@
 UNPROVIDED_CACHE_IDENTIFIER = "_unprovided"
 
 # Async settings
-DEFAULT_FUTURE_TIMEOUT = 120
+DEFAULT_FUTURE_TIMEOUT = 900000  # candle downloader needs more time
```

Comment:
I don't think we should overwrite this value this way as it can get people's bot completely stuck. If we really need to increase it, I thing we should get this value from an env variable like this:
`os.getenv("DEFAULT_FUTURE_TIMEOUT", 120)`

Reply:
a env would work for me, but if anyone wants to download data on 5 min chart and below it will fail

Reply:
Is there a value you think would be more suitable ?

Reply:
could be multiple hours. I remember 5 minute data alone on bitstamp took ~half an hour

Reply:
Then we will add a parameter to allow long actions

Reply:
I'm setting it to 1h for data collector

Reply:
yeah thats the only time out that affects others. 2h should be fine + an option is great